cudamapper/src/application_parameters.cpp

```diff
@@ -283,22 +283,22 @@ void ApplicationParameters::help(int32_t exit_code)
             length of batch sized used for target in MB [30])"
               << R"(
         -F, --filtering-parameter
-            filter all representations for which sketch_elements_with_that_representation/total_sketch_elements >= filtering_parameter), filtering disabled if filtering_parameter == 1.0 [1'000'000'001] (Min = 0.0, Max = 1.0))"
+            Remove representations with frequency (sketch_elements_with_that_representation/total_sketch_elements) >= filtering_parameter. Filtering is disabled if filtering_parameter == 1.0 [1'000'000'001] (Min = 0.0, Max = 1.0, default=1e-5))"
```

Comment:
I know this is my code, but even I don't remember what's `[query_indices_being_generated]`, so I would suggest removing it.
As in other lines `[]` is used for default value I would use it here as well